bluesky/traffic/route.py

```diff
@@ -422,224 +480,112 @@ def addwpt(self, iac, name, wptype, lat, lon, alt=-999., spd=-999., afterwp="",
         # For safety
         self.nwp = len(self.wplat)
 
+        name = name.upper().strip()
+
+        wplat = lat
+        wplon = lon
+

```

Comment:
Giving default values for these (wplat, wplon) and wpok saves a lot of cases below.